llvm/lib/Transforms/Vectorize/LoopVectorizationPlanner.h

```diff
@@ -150,11 +151,11 @@ class VPBuilder {
   /// its underlying Instruction.
   VPInstruction *createNaryOp(unsigned Opcode, ArrayRef<VPValue *> Operands,
                               Instruction *Inst = nullptr,
+                              const VPIRMetadata &MD = {},
```

Comment:
Maybe good to be consistent and also add a similar default parameter to `createInstruction`?